src/core/BufferGeometry.js

```diff
@@ -101,13 +101,13 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 	},
 
-	addGroup: function ( start, count, materialIndex ) {
+	addGroup: function ( start, count, materialIndex = 0 ) {
 
 		this.groups.push( {
 
-			start: start,
-			count: count,
-			materialIndex: materialIndex !== undefined ? materialIndex : 0
+			start,
+			count,
+			materialIndex
```

Comment:
Curious to see how Babel will handle this 🤓

Reply:
Babel should handle this just fine, no? It's valid JS.

Reply:
Yep!